app/controllers/bucket_jobs_controller.rb

```diff
@@ -22,16 +23,28 @@ def destroy
   end
 
   private
+    def create_job_application
+    @job_application = @job.job_applications.create(summary: "Please write a brief paragraph explaining why you would be a good fit for this job.")
+    @job_application.update(user_id: current_user.id) if current_user
```

Comment:
Line 27 creates and 28 updates which is two DB queries. Let's try to get this done with one. I think something like this should work.

```ruby
@job_application = @job.job_applications.create(
  summary: "Please write a brief paragraph explaining why you would be a good fit for this job.",
  user: current_user
)
```